### PR TITLE
ci: create crowdin action

### PR DESCRIPTION
**Pull request by sy-records**
Translations can now be done at https://translate.docsifyjs.org

ref: #2432

### CODE REVIEW

**Review by paulhibbitts — APPROVED** [review]
GitHub action looks good to me, thanks for this PR @sy-records !

**Review by jhildenbiddle — DISMISSED** [review]
LGTM.

File: src/core/render/emoji-data.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
- 
+/* eslint-disable */
```
Comment (jhildenbiddle) [inline src/core/render/emoji-data.js:1]: Looks like this file was checked in by accident. No need to include the updated emoji short codes with this PR.

**Review by jhildenbiddle — CHANGES_REQUESTED** [review]

File: src/core/render/emoji-data.js (hunk 1 above)
Comment (sy-records) [reply src/core/render/emoji-data.js:1]: I merged it here on purpose, I wanted to modify the emoji action.

This file does not require `/* eslint-disable */`?

Comment (jhildenbiddle) [reply src/core/render/emoji-data.js:1]: I believe this is correct now that `eslint-data.js` is ignored via `/eslint.config.js`. The comment is being added by the `build/emoji.js` script while auto-generating `emoji-data.js` content.

**Pull request merged**